src/aleph/vm/models.py

```diff
@@ -130,7 +130,11 @@ def uses_payment_stream(self) -> bool:
     @property
     def has_resources(self) -> bool:
         assert self.vm, "The VM attribute has to be set before calling has_resources()"
-        return self.vm.resources_path.exists() if self.hypervisor == HypervisorType.firecracker else True
+        if isinstance(self.vm, (AlephFirecrackerExecutable, AlephFirecrackerInstance)):
```

Comment:
Also is needed to add the class `AlephFirecrackerProgram`

Reply:
Hum... in fact maybe I should just remove `AlephFirecrackerInstance` from the `if`, since they are both child classes of `AlephFirecrackerExecutable`.